src/Microsoft.FeatureManagement/FeatureManagementError.cs

```diff
@@ -21,6 +21,16 @@ public enum FeatureManagementError
         /// <summary>
         /// A feature that was requested for evaluation was not found.
         /// </summary>
-        MissingFeature
+        MissingFeature,
+
+        /// <summary>
+        /// A configuration had conflicting values.
```

Comment:
The error code isn't limited to configuration. It could be any conflict we encounter in our feature management system.

Reply:
This was resolved, but I don't see any reply or related change.

Reply:
My fault, didn't save it in visual studio so it wasn't included in the commit. Added it now. I went with: `There was a conflict in the feature management system.`